streams/src/main/java/org/apache/kafka/streams/processor/internals/InternalTopologyBuilder.java

```diff
@@ -532,14 +539,14 @@ public final void addStateStore(final StoreBuilder<?> storeBuilder,
         nodeGroups = null;
     }
 
-    public final <KIn, VIn, KOut, VOut> void addGlobalStore(final StoreBuilder<?> storeBuilder,
+    public final <KIn, VIn> void addGlobalStore(final StoreBuilder<?> storeBuilder,
```

Comment:
Why do we drop KOut/VOut?

Reply:
I didn't realize it in the last PR, but the output types for the `stateUpdate` processor is always `Void, Void` (ie., you can't forward anything, since the only function of that processor is to update the store itself). So we don't need the parameters; I've just expanded them to `Void, Void` below.